packages/core/admin/admin/src/components/tests/Widget.test.tsx

```diff
@@ -1,14 +1,15 @@
 import { Cog } from '@strapi/icons';
 import { render, screen } from '@tests/utils';
 
-import { Widget } from '../Widget';
+import { WidgetRoot } from '../../pages/Home/HomePage';
```

Comment:
@remidej I forgot to update the tests. I also moved them to be in the components folder with the WidgetHelper. I don't like that I have to export the WidgetRoot from the homepage just for this test but currently don't have a better solution. Maybe it's nbd 🤷 

p.s.

sorry had to force push along the way

Reply:
Probably not a big deal, thanks for the heads up